x-pack/plugins/rollup/server/routes/api/jobs.js

```diff
@@ -45,6 +45,10 @@ export function registerJobsRoute(server) {
           ...rest
         } = request.payload.job;
 
+        console.log('--------------------------------');
```

Comment:
Are you ready to remove these? If you want to keep them then please move them to use the server logger and being a little less verbose, combined into a single line, etc.

Reply:
Yeah all the logs will be gone no worries 😊